mod-finance/examples/transaction_allocation.sample

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "ee683f15-b1e1-40c8-866d-4e70da03c723",
+  "amount": 25,
+  "currency": "USD",
+  "description": "PO_Line: History of Incas",
+  "fiscalYearId": "4438bc3b-4b4a-4e5e-9886-5f69b48a7a32",
+  "fromFundId": "27f27438-3197-4281-a353-511829421d6e",
+  "paymentEncumbranceId": "80c28fb4-c34e-48d6-b8c2-de6660c3e0aa",
+  "source": "Voucher",
+  "sourceFiscalYearId": "01705ce5-3c3d-4c30-b23d-a36feec28259",
+  "sourceInvoiceId": "b360db6c-b825-4d74-ac63-5be506892e2b",
```

Comment:
As far as I understand, the `sourceInvoiceId` and `sourceInvoiceLineId` fields will be used only for Payment and Credit.